.gitignore

```diff
@@ -52,3 +52,5 @@ junit.xml
 
 
 ganache-deployments.json
+ganache-deployments*.json
+.ganache-deployment-cache/
```

Comment:
What folder does this get put into? Maybe if it's only in `packages/devtools` we can put this in `packages/devtools/.gitignore`.

Reply:
It's in the root of the monorepo.